tests/Feature/Engine/ErrorEndEventTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Tests\Feature\Engine;
+
+use ProcessMaker\Nayra\Contracts\Bpmn\ActivityInterface;
+use ProcessMaker\Nayra\Contracts\Bpmn\ErrorEventDefinitionInterface;
+use ProcessMaker\Nayra\Contracts\Bpmn\EventInterface;
+use ProcessMaker\Nayra\Contracts\Bpmn\ProcessInterface;
+use ProcessMaker\Repositories\BpmnFileRepository;
+
+/**
+ * Test an error end event.
+ *
+ */
+class ErrorEndEventTest extends EngineTestCase
+{
+
+    /**
+     * Test a global Error End Event
+     *
+     */
+    public function testErrorEndEventTopLevel()
+    {return;
```

Comment:
This function is returning at the beginning, so it is not executed...

Reply:
Fixed